apps/web/composables/useHomepage/types.ts

```diff
@@ -1,19 +1,51 @@
-import { HeroContentProps } from '~/components/ui/HeroCarousel/types';
-import { MediaItemProps } from '~/components/ui/MediaCard/types';
+interface ImageProps {
+  lg: string;
+  md: string;
+  sm: string;
+  xs: string;
+}
 
-export type Featured = {
-  headline: string;
-  categoryId: string;
-};
+interface HeroContentProps {
+  image: ImageProps;
+  tagline: string;
+  taglineColor: string;
+  heading: string;
+  headingColor: string;
+  description: string;
+  alt: string;
+  descriptionColor: string;
+  callToAction: string;
+  link: string;
+}
 
-export interface HomeData {
+interface UiHeroCarouselOptions {
   hero: HeroContentProps[];
-  mediaCard: MediaItemProps[];
-  featured: Featured[];
+}
+
+interface UiMediaCardOptions {
+  text: string;
+  image: string;
+  alt: string;
+  alignment: string;
+}
+
+interface ProductRecommendedProductsOptions {
```

Comment:
**todo**: Import from the existing type instead. If we want to have a new interface for naming purposes, that's fine, but it should just wrap the type in the end.

Reply:
Will be addressed separately.